pkg/horizon/const.go

```diff
@@ -38,9 +38,27 @@ const (
 	// HorizonExtraVolTypeUndefined can be used to label an extraMount which is
 	// not associated to anything in particular
 	HorizonExtraVolTypeUndefined storage.ExtraVolType = "Undefined"
+
 	// Horizon is the global ServiceType that refers to all the components deployed
 	// by the horizon-operator
 	Horizon storage.PropagationType = "Horizon"
+
+	//LogFile -
+	LogFile = "/var/log/horizon/horizon.log"
```

Comment:
Are we also planning to capture Apache logs as part of this? Or do you feel that just the Horizon application logs are sufficient for any support / debugging requirements your team may have?

Current logging for Apache is just to the stdout of the container, so that might be sufficient:
https://github.com/openstack-k8s-operators/horizon-operator/blob/main/templates/horizon/config/httpd.conf#L25-L26

It just means logs will be lost when / if the pod is evicted from a node.

This can probably be a separate PR and topic, but just asking to make sure it has been considered.

Reply:
The other thing about writing it to a specific log file is that we wont be able to see them simply by `oc logs` on the Horizon pod. So we will also need to add a sidecar container to the pod which will just run `tail -f /var/log/horizon/horizon.log`. That architecture is defined under this section:
https://kubernetes.io/docs/concepts/cluster-administration/logging/#sidecar-container-with-logging-agent

We can just add a new container to the pod called `horizon-logs` or something. That way, users will be able to clearly tell which container they can check to get Horizon Django application logs.